orchestra/migrations/0083_update_staffbot_request_status_choices.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.2.9 on 2020-03-30 19:21
+
+from django.db import migrations, models
+
+def migrate_request_status(apps, schema_editor):
+    StaffBotRequest = apps.get_model('orchestra', 'StaffBotRequest')
+    StaffingResponse = apps.get_model('orchestra', 'StaffingResponse')
+    StaffingRequestInquiry = apps.get_model('orchestra', 'StaffingRequestInquiry')
+
+    DONE_SENDING_INQUIRIES = 1
+    CLOSED = 2
+
+    requests = (StaffBotRequest.objects.all()
+                .exclude(status=CLOSED))
+    for request in requests:
+        responses = StaffingResponse.objects.filter(
+                        request_inquiry__request=request)
+        has_winner = responses.filter(is_winner=True).exists()
+        if has_winner:
+            request.status = CLOSED
+        elif request.status == DONE_SENDING_INQUIRIES:
+            inquiries = (
```

Comment:
sanity-checking that this is a copy/paste of the latest iteration on your `check_responses_complete` code

Reply:
Yes, it is.